tests/cypress/support/stateSubmissionFormCommands.ts

```diff
@@ -274,19 +289,28 @@ Cypress.Commands.add('fillOutNewRateCertification', () => {
     cy.findAllByTestId('errorMessage').should('have.length', 0)
 })
 
-Cypress.Commands.add('fillOutAmendmentToPriorRateCertification', () => {
+Cypress.Commands.add('fillOutAmendmentToPriorRateCertification', (id = 0) => {
     // Must be on '/submissions/:id/edit/rate-details'
     // Must be a contract and rates submission
     cy.wait(2000)
     cy.findByText('Amendment to prior rate certification').click()
     cy.findByText(
         'Certification of capitation rates specific to each rate cell'
     ).click()
+
     cy.wait(2000)
-    cy.findAllByLabelText('Start date').eq(0).type('02/29/2024')
-    cy.findAllByLabelText('End date').eq(0).type('02/28/2025')
-    cy.findAllByLabelText('Start date').eq(1).type('03/01/2024')
-    cy.findAllByLabelText('End date').eq(1).type('03/01/2025')
+
+/* 
+    There are currently multiple date range pickers on the page with the same label names (start date, end date) and different headings
+    Preferred approach would be targeting by via findBy* or a custom data-cyid attribute
+    However, surfacing custom attributes on the nested inputs in third party component DateRangePicker not possible in current react-uswds version 
+    For now using targeting by html id (anti-pattern)
+*/
+    cy.get(`[id="rateInfos.${id}.rateDateStart"]`).type('02/01/2023')
```

Comment:
is there a parent div or something we could use to use .within here to differentriate between? Or is that only in testing library itself? No worries if not but that’s my only thought.

Reply:
@macrael Yeah, good point. Looking at this code again, maybe what I should do is put the `data-cyid` attribute I was struggling to get on the date inputs on the overarching form group div instead. Then use `within`.  Without a code change there isn't anything unique about the parent since its just a generic form group. 

I'm thinking to merge this as is but will keep looking at things since I'm going to do more uswds upgrade stuff next